Multisig/UI/Transactions/TransactionDetails/TransactionDetailsViewModel.swift

```diff
@@ -89,7 +89,10 @@ class TransactionDetailsViewModel: BasicLoadableViewModel {
 
     func sign(safeAddress: Address) {
         guard let transferTx = transactionDetails as? TransferTransactionViewModel,
-              let transaction = transferTx.transaction else { return }
+              let transaction = transferTx.transaction else {
+            assertionFailure("We have a technical problem. You need to restart the app.")
```

Comment:
`assertionFailure("Expected to get transfer view model and transaction")`

Note that assertionFailure is only triggered in development builds, so this is a developer-facing message

Reply:
done